modules/compute/instance.tf

```diff
@@ -4,35 +4,53 @@
 data "oci_core_subnet" "subnet" {
   subnet_id = var.subnet_id
 }
+
 data "oci_core_vcn" "vcn" {
   vcn_id = local.vcn_id
 }
+
 locals {
   compartment_id = data.oci_core_vcn.vcn.compartment_id
 }
-locals {
-  hostname_label = replace(lower(var.display_name), " ", "")
-  assign_public_ip = data.oci_core_subnet.subnet.prohibit_public_ip_on_vnic?false:true
-  vcn_id = data.oci_core_subnet.subnet.vcn_id
-  ad_names = keys(var.fd_map)
-  num_ads = length(var.fd_map)
 
+locals {
+  hostname_label   = replace(lower(var.display_name), " ", "")
+  assign_public_ip = data.oci_core_subnet.subnet.prohibit_public_ip_on_vnic ? false : true
+  vcn_id           = data.oci_core_subnet.subnet.vcn_id
+  ad_names         = keys(var.fd_map)
+  num_ads          = length(var.fd_map)
+  compute_flexible_shapes = [
+    "VM.Standard.E3.Flex",
+    "VM.Standard.E4.Flex",
+    "VM.Standard.A1.Flex",
```

Comment:
Did you test on ARM shapes?

Reply:
It works fine with A1:

1. Console data - jboss0 instance with A1
![image](https://user-images.githubusercontent.com/19390609/132481134-8a90f4ba-3cbe-4f0b-83b2-39efdb93abaf.png)

2. Console data - LB
![image](https://user-images.githubusercontent.com/19390609/132481589-49c26192-20a6-4bab-858e-ed134788b48d.png)

3. Wildfly access via LB IP
![image](https://user-images.githubusercontent.com/19390609/132481291-1ff531d8-5845-4d73-bf7d-d7123b556bdb.png)

Reply:
great, thanks!